plugins/QnA/class.qna.plugin.php

```diff
@@ -1113,10 +1113,21 @@ public function postController_afterForms_handler($sender) {
      * @param PostController $sender Sending controller instance.
      */
     public function postController_question_create($sender, $categoryUrlCode = '') {
-            // Create & call PostController->discussion()
-            $sender->View = PATH_PLUGINS . '/QnA/views/post.php';
-            $sender->setData('Type', 'Question');
-            $sender->discussion($categoryUrlCode);
+        $categoryModel = new CategoryModel();
+        if ($categoryUrlCode != '') {
+            $category = (array)$categoryModel->getByCode($categoryUrlCode);
+            $category = $categoryModel::permissionCategory($category);
+            $isAllowedTypes = isset($category['AllowedDiscussionTypes']);
+            $isAllowedQuestion = in_array('Question', $category['allowedDiscussionType']);
+        }
```

Comment:
I think you might have casing error here.  
$category['allowedDiscussionType'] vs $category['AllowedDiscussionTypes']

Reply:
Good catch. It was getting set to false and because of the wrong key